### PR TITLE
Create docker image with service

### PR DESCRIPTION
**Pull request by logachev**

### CODE REVIEW

**Comment by aaron-schnieder** [comment]
@logachev just to double check, all of the deleted files are the full .net framework versions correct?

File: services/CertificateGeneration/CertificateGenerationTests.NetCore/Wrappers/CertificatesWrapperTests.cs [hunk 1]
```diff
@@ -60,11 +58,30 @@ public void GenerateCertificateTest_InvalidSubjectNameThrowsException()
...
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                // When runs on Windows, it throws different exception.
+                try
+                {
+                    certificatesWrapper.GenerateCertificate(properties);
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e.Message);
```
Comment (aaron-schnieder) [inline services/CertificateGeneration/CertificateGenerationTests.NetCore/Wrappers/CertificatesWrapperTests.cs]: Remove console.WriteLine (I assume this was in there for debugging)

File: services/CertificateGeneration/DockerFile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+FROM microsoft/dotnet:2.0-sdk AS build
```
Comment (aaron-schnieder) [inline services/CertificateGeneration/DockerFile]: I think this should be the .netcore runtime, not SDK. The SDK is going to cause the docker image to be huge. https://hub.docker.com/r/microsoft/dotnet/

File: services/CertificateGeneration/DockerFile [hunk 3]
```diff
@@ -0,0 +1,21 @@
+FROM microsoft/dotnet:2.0-sdk AS build
+WORKDIR /app
+
+COPY *.sln .
+COPY CertificateGeneration.NetCore/*.csproj ./CertificateGeneration.NetCore/
+COPY CertificateGenerationTests.NetCore/*.csproj ./CertificateGenerationTests.NetCore/
```
Comment (aaron-schnieder) [inline services/CertificateGeneration/DockerFile]: Do you need the tests in the docker image?

**Review by aaron-schnieder — CHANGES_REQUESTED** [review]

File: services/CertificateGeneration/DockerFile (hunk 3 above)
Comment (logachev) [reply services/CertificateGeneration/DockerFile]: Yes for build image. Docker runs unittests (see line 16-18), if they fail docker build fails.

**Review by aaron-schnieder — APPROVED** [review]

**Pull request merged**